.github/workflows/kata-deploy.yaml

```diff
@@ -0,0 +1,34 @@
+on: push
+
+name: Build, Test, and Publish kata-deploy
+
+jobs:
+  kata-deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: build-and-push-kata-deploy-ci
+        run: |
+          git clone https://github.com/amshinde/kata-packaging
```

Comment:
Eek - personal GitHub account! Can this be changed please?

Reply:
this file isn't intended to be merged. It's there as an FYI (see the commits themselves)